build/Build.cs

```diff
@@ -291,6 +296,45 @@ public Build()
                           CopyFile(file, LocalPackagesDirectory / Path.GetFileName(file), FileExistsPolicy.Overwrite);
                   });
 
+        Target CopySashimiPackagesForConsolidation =>
+            _ => _.DependsOn(Compile)
+                 .Executes(() =>
+                           {
+                               foreach (var file in Directory.GetFiles(SashimiPackagesDirectory))
+                                   CopyFile(file, ArtifactsDirectory / Path.GetFileName(file), FileExistsPolicy.Overwrite);
+                           });
+
+        Target PackageConsolidatedCalamariZip =>
+            _ => _.DependsOn(PackBinaries)
```

Comment:
The dependency tree here runs the `Clean` target, which removes everything from `**/obj`, `**/bin`, `./artifacts` and `./publish`.

We may want to look at keeping the Consolidated Calamari package around, because the Consolidate task firstly looks to see _whether_ it needs to rebuild the package, and if not, skips it. By clearing it away every time, we lose this potential efficiency.

Reply:
Yep. This is simple to do - I'll add this too, in the follow-up with cleanups and optimisations PR.